dm_control/utils/inverse_kinematics.py

```diff
@@ -46,12 +46,13 @@ def qpos_from_site_pose(physics,
                         max_update_norm=2.0,
                         progress_thresh=20.0,
                         max_steps=100,
-                        inplace=False):
+                        inplace=False,
+                        null_space_method=True):
```

Comment:
What do you think about not using a boolean variable, and instead having an enum for regularization_method that can be NULLSPACE_METHOD or DAMPED_LEAST_SQUARES? That would allow us to add more methosd in the future without breaking the API.

BTW: I think it's confusing that there's a function called nullspace_method and a variable called null_space_method. If we stick with the boolean, name the variable use_nullspace_method.

Same below.